MdePkg/Library/BaseLib/X86UnitTestHost.c

```diff
@@ -66,6 +66,15 @@ UnitTestHostBaseLibAsmCpuid (
   OUT     UINT32  *Edx   OPTIONAL
   )
 {
+  UINT32  RetEcx;
+
+  RetEcx = 0;
+  switch (Index) {
+    case 1:
+      RetEcx |= BIT30; /* RdRand */

```

Comment:
I don't think it is right fix to hardcode the value.

I recommend calling CPUID to get real value.

Reply:
Before merging https://github.com/tianocore/edk2/pull/5714 the situation was exactly the same, the host test simply assumed rdrand support is there without actually checking.  So, yes, it's not ideal, but it also is not worse than it used to be.

Note that proper rdrand checking needs more than just calling CPUID to figure the real value, the test cases also need an update to either skip tests or fallback to another RNG source in case rdrand is not there.

Given that rdrand is roughly a one decade old (2012+ for intel, 2015+ for amd) and using rdrand without checking apparently was not much of an problem before I'm not convinced this would be worth the effort.